src/game.cpp

```diff
@@ -5279,6 +5277,41 @@ void game::draw_sidebar()
     draw_sidebar_messages();
 }
 
+void game::draw_safe_mode()
```

Comment:
Should expect a `WINDOW *, int y, int x` as arguments as opposed to hardcoding them